config/default.yml

```diff
@@ -929,7 +929,9 @@ RSpec/FactoryBot/ConsistentParenthesesStyle:
   SupportedStyles:
     - require_parentheses
     - omit_parentheses
+  ExplicitOnly: false
```

Comment:
I was thinking whether this could be applied to `RSpec/FactoryBot` (the whole department) and affect the matcher on the receiver (whether to include nil receiver or not). @pirj WDYT?

Reply:
Indeed it is, looks good. So we cut out `ExplicitOnly` into a common class to make it easier to handle, and added the configuration values to the other cop. WDYT?